locale/zh_CN/LC_MESSAGES/client.po

```diff
@@ -1923,483 +2062,223 @@ msgstr "使用应用程序配对"
+#~ msgstr "上次同步 %(translatedTimeAgo)s 在 %(translatedCountry)s %(translatedState)s %(translatedCity)s 附近"
 
-#: .es5/templates/support.mustache:3 .es5/templates/support.mustache:6
-msgid "What do you need help with?"
-msgstr "您需要什么帮助？"
+#~ msgid "Last sync over %(translatedTimeAgo)s near %(translatedCity)s, %(translatedState)s, %(translatedCountry)s"
+#~ msgstr "上次同步早于 %(translatedTimeAgo)s 在 %(translatedCountry)s %(translatedState)s %(translatedCity)s 附近"
 
-#: .es5/templates/support.mustache:4 .es5/models/support-form.js:25
-msgid "General inquiries"
-msgstr "常规查询"
+#~ msgid "Last sync time unknown"
+#~ msgstr "上次同步时间未知"
 
-#: .es5/templates/support.mustache:5 .es5/models/support-form.js:25
-msgid "Payment & billing"
-msgstr "支付和账单"
+#~ msgid "Subscribed successfully"
+#~ msgstr "订阅成功"
 
-#: .es5/templates/support.mustache:6 .es5/models/support-form.js:25
-msgid "Connection issues"
-msgstr "连接问题"
+#~ msgid "Open tabs"
+#~ msgstr "打开的标签页"
 
-#: .es5/templates/support.mustache:7 .es5/models/support-form.js:25
-msgid "Getting started"
-msgstr "开始使用"
+#~ msgid "Logins"
+#~ msgstr "登录信息"
 
-#: .es5/templates/support.mustache:8 .es5/models/support-form.js:25
-msgid "Account issues"
-msgstr "账户问题"
+#~ msgid "Credit cards"
+#~ msgstr "信用卡"
 
-#: .es5/templates/support.mustache:9 .es5/templates/support.mustache:8
-msgid "Subject (optional)"
-msgstr "主题（选填）"
+#~ msgid "Accounts with two-step authentication do not support pairing at this time"
+#~ msgstr "暂不支持与开启两步验证的账户配对"
 
-#: .es5/templates/support.mustache:10
-msgid "Description of Issue"
-msgstr "问题描述"
+#~ msgid "Please try again later"
+#~ msgstr "请稍后再试"
 
-#: .es5/templates/settings/delete_account.mustache:5 .es5/templates/settings/delete_account.mustache:7
-msgid "You've connected your Firefox account to Mozilla products that keep you secure and productive on the web:"
-msgstr "您已将 Firefox 账户关联至可让您畅游网络更安全、更高效的 Mozilla 产品："
+#~ msgid "Service unavailable, try again soon"
+#~ msgstr "服务不可用，请稍候重试"
 
-#: .es5/templates/settings/delete_account.mustache:14 .es5/templates/settings/delete_account.mustache:16
-msgid "Please acknowledge that by deleting your account:"
-msgstr "请确认，若您真要删除账户："
+#~ msgid "%(email)s registered"
+#~ msgstr "%(email)s 已注册"
 
-#: .es5/templates/settings/delete_account.mustache:15
-msgid "Any subscriptions you have will be cancelled"
-msgstr "将取消您的所有订阅内容"
+#~ msgid "Choose what to sync"
+#~ msgstr "选择要同步的内容"
 
-#: .es5/templates/settings/delete_account.mustache:16 .es5/templates/settings/delete_account.mustache:18
-msgid "You may lose saved information and features within Mozilla products"
-msgstr "您可能丢失在 Mozilla 产品中保存的信息以及功能"
+#~ msgid "Save selections"
+#~ msgstr "保存选择"
 
-#: .es5/templates/settings/delete_account.mustache:17 .es5/templates/settings/delete_account.mustache:19
-msgid "Reactivating with this email may not restore your saved information"
-msgstr "用此电子邮件地址重新激活，可能无法恢复您已保存的信息"
+#~ msgid "Create an account"
+#~ msgstr "创建账户"
```

Comment:
Is the mass commenting out of translations here intended?

Some strings are currently unlocalized in prod FxA. This one can be seen at https://accounts.firefox.com/signin .